requirements/pytorch/loggers.txt

```diff
@@ -1,8 +0,0 @@
-# NOTE: the upper bound for the package version is only set for CI stability, and it is dropped while installing this package
-#  in case you want to preserve/enforce restrictions on the latest compatible version, add "strict" as an in-line comment
-
-# all supported loggers
-neptune-client>=0.10.0, <0.16.10
```

Comment:
so we never test loggers?

Reply:
We've never tested loggers

Reply:
Can we have some integration test for this or remove loggers completely from the codebase :(

Reply:
We've been fine without installing the loggers in CI, the tests all use mocks. We don't need to remove them

Reply:
but we do not test it with real package, right so how we know it still works?

Reply:
In Lightning, a logger is a wrapper around a third-party logger. Obviously it is not on us to test the third-party behaves correctly. However, it is our responsibility to test that our wrapper calls the third-party objects correctly. This is sufficient for the vast majority of testing cases.